fineract-provider/src/main/java/org/apache/fineract/notification/data/NotificationData.java

```diff
@@ -0,0 +1,220 @@
+
+    private Long id;
+    private Long userId;
+    private String objectType;
+    private Long objectId;
+    private String action;
+    private String actor;
+    private String content;
+    private boolean isSystemGenerated;
+    private String tenantIdentifier;
+    private String createdAt;
+    private Long officeId;
+    private List<Long> userIds;
+
+    public NotificationData() {
+
+    }
+
+    public NotificationData(NotificationBuilder builder) {
+        this.userId = builder.userId;
+        this.objectType = builder.objectType;
+        this.objectId = builder.objectId;
+        this.action = builder.action;
+        this.actor = builder.actor;
+        this.content = builder.notificationContent;
+        this.isSystemGenerated = builder.isSystemGenerated;
+        this.tenantIdentifier = builder.tenantIdentifier;
+        this.userIds = builder.userIds;
+        this.officeId = builder.officeId;
+    }
+
+    public Long getOfficeId() {
+        return officeId;
+    }
+
+    public void setOfficeId(Long officeId) {
+        this.officeId = officeId;
+    }
+
+    public List<Long> getUserIds() {
+        return userIds;
+    }
+
+    public void setUserId(List<Long> userIds) {
+        this.userIds = userIds;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    public void setCreatedAt(String createdAt) {
+        this.createdAt = createdAt;
+    }
+
+    public Long getUserId() {
+        return userId;
+    }
+
+    public void setUserId(Long userId) {
+        this.userId = userId;
+    }
+
+    public String getObjectType() {
+        return objectType;
+    }
+
+    public void setObjectType(String objectType) {
+        this.objectType = objectType;
+    }
+
+    public Long getObjectIdentfier() {
+        return objectId;
+    }
+
+    public void entifier(Long objectIdentifier) {
+        this.objectId = objectIdentifier;
+    }
+
+    public String getAction() {
+        return action;
+    }
+
+    public void setAction(String action) {
+        this.action = action;
+    }
+
+    public String getActor() {
+        return actor;
+    }
+
+    public void setActor(String actor) {
+        this.actor = actor;
+    }
+
+    public String getContent() {
+        return content;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    public boolean isSystemGenerated() {
+        return isSystemGenerated;
+    }
+
+    public void setSystemGenerated(boolean systemGenerated) {
+        isSystemGenerated = systemGenerated;
+    }
+
+    public String getTenantIdentifier() {
+        return tenantIdentifier;
+    }
+
+    public void setTenantIdentifier(String tenantIdentifier) {
+        this.tenantIdentifier = tenantIdentifier;
+    }
+
+    public static class NotificationBuilder {
```

Comment:
Hi,
The use of this static class is only in `NotificationData` class. I have used this static inner class to implement a builder pattern. Is it advisable to move to another file ?